docs/source/intro.rst

```diff
@@ -0,0 +1,72 @@
+.. _intro:
+
+Introduction: why is it complex to assess DEM accuracy and precision?
+=====================================================================
+
+Digital Elevation Models are numerical representations of elevation. They are generated from different instruments (e.g., radiometer, radar, lidar), acquired in different conditions (e.g., ground, airborne, satellite), and using different post-processing techniques (e.g., stereophotogrammetry, interferometry).
+
+While some complexities are specific to certain instruments and methods, all DEMs generally have:
+
+- an **arbitrary Ground Sampling Distance (GSD)** that does not necessarily represent their underlying spatial resolution,
```

Comment:
To an unexperienced reader (for which this is targeted, right?) GSD and spatial resolution may be interpreted to be the same. Perhaps add a link to a source that differs the two?

Reply:
I would also remove the term "arbitrary", as the choice of the resolution is not fully arbitrary. If it's a standard product, the resolution still gives an idea on what the actual resolution is (although these days resolution mostly depends on the access plan, e.g. for TDX, 90 m is free, 12 m is for purchase).